mw-webapp/src/component/progressBar/ProgressBar.module.scss

```diff
@@ -19,7 +20,11 @@ $progressBarBorderRadius: 20px;
 }
 
 .progressIndicatorPercent {
+  position: absolute;
+  right: 0;
+  left: 0;
```

Comment:
how does it works at the same time?)

Reply:
I can't explain exactly how it works). But this bundle makes it possible to set text-align: center.

Reply:
Hahah) ok I see